### PR TITLE
fix: secureboot pxe

### PR DESCRIPTION
**Pull request by frezbo**
Fix secureboot pxe script.

Fixes: #176
Fixes: #https://github.com/siderolabs/talos/issues/9947

### CODE REVIEW

File: internal/frontend/http/pxe.go [hunk 1]
```diff
@@ -92,6 +80,20 @@ func (f *Frontend) handlePXE(ctx context.Context, w http.ResponseWriter, _ *http
...
+	if prof.SecureBootEnabled() {
+		return ensure.Value(template.New("secureboot.ipxe").
+			Parse(securebootIPXE)).
+			Execute(w,
+				struct {
+					UKIURL  string
+					Cmdline string
+				}{
+					UKIURL:  f.options.ExternalPXEURL.JoinPath("image", schematicID, versionTag, fmt.Sprintf("%s-%s-secureboot-uki.efi", prof.Platform, prof.Arch)).String(),
+					Cmdline: string(cmdline),
```
Comment (smira) [inline internal/frontend/http/pxe.go:92]: I'm a bit confused about this one - if SecureBoot is enabled already, UKI should ignore the cmdline passed?

Comment (smira) [reply internal/frontend/http/pxe.go:92]: Adding cmdline won't hurt, but still not sure I fully understand the flow

Comment (frezbo) [reply internal/frontend/http/pxe.go:92]: well with pxe it just boots the uki as normal EFI PE executable, UKI command line is parsed if using sd-boot/sd-stub combo, pxe bypasses that

**Review by smira — APPROVED** [review]

**Pull request merged**